# try:
#     x = 1 / 0
# except ZeroDivisionError as ex_:
#     print(ex_)
#
# print("Hello World!")

# try:
#     raise ValueError
# except BaseException as ex_:
#     print(type(ex_).__name__)

raise ValueError("Вы ввели ошибочное значение!!")